frontend/benefit/applicant/src/components/applications/forms/application/step1/ApplicationFormStep1.tsx

```diff
@@ -149,19 +150,33 @@ const ApplicationFormStep1: React.FC<DynamicFormStepComponentProps> = ({
           </StyledFormGroup>
         )}
         <StyledFormGroup>
-          <TextInput
-            id={fields.companyIban.name}
-            name={fields.companyIban.name}
-            label={fields.companyIban.label}
-            placeholder={fields.companyIban.placeholder}
-            onChange={formik.handleChange}
-            onBlur={formik.handleBlur}
+          <InputMask
+            mask={fields.companyIban.mask?.format ?? ''}
+            maskChar={null}
             value={formik.values.companyIban}
-            invalid={!!getErrorMessage(APPLICATION_FIELDS.COMPANY_IBAN)}
```

Comment:
we need then to figure out how to apply the validation with hds like error message

Reply:
Yeah, I was wondering whether to implement validation in the scope of this PR or not 🤔

Reply:
Perhaps better to do the validation as well to see that it plays well with hds TextInput ?

Reply:
Added validation.